src/css/mobile.styl

```diff
@@ -14,6 +14,12 @@
 		right 300px
 		top 70px
 
+	.esriPopupWrapper
+		.sizer
```

Comment:
Sizer at 100% is too narrow!
![screen shot 2017-02-13 at 5 03 11 pm](https://cloud.githubusercontent.com/assets/5713523/22905466/54d44468-f20e-11e6-9768-493a43b0b1bf.png)